examples/torch/classification/main.py

```diff
@@ -727,19 +728,19 @@ def validate(val_loader, model, criterion, config, epoch=0, log_validation_info=
                     )
                 )
 
-        if is_main_process() and log_validation_info:
-            config.tb.add_scalar("val/loss", losses.avg, len(val_loader) * epoch)
-            config.tb.add_scalar("val/top1", top1.avg, len(val_loader) * epoch)
-            config.tb.add_scalar("val/top5", top5.avg, len(val_loader) * epoch)
-            config.mlflow.safe_call("log_metric", "val/loss", float(losses.avg), epoch)
-            config.mlflow.safe_call("log_metric", "val/top1", float(top1.avg), epoch)
-            config.mlflow.safe_call("log_metric", "val/top5", float(top5.avg), epoch)
+        if is_main_process():
+            if log_validation_info:
```

Comment:
Keep the original version with `and` to have less levels of indent

Reply:
Done